### PR TITLE
Update UG with 3 find-tag-related new features in contact list.

### PR DESCRIPTION
**Pull request by yanbingtao**
- `c-today`
- `c-tomorrow`
- `c-tag-find KEYWORD [KEYWORDS]`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -299,7 +324,24 @@ Examples:
...
+
+Finds all contacts that contain the KEYWORD(s) in their tags.
+
+Format: `c-tag-find KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `Friday` will match `friday`.
+* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
+* Only the tag names are searched.
+* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
+* Employees whoes tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
```
Comment (WM71811) [inline docs/UserGuide.md]: I think whoes is a typo?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -299,7 +324,24 @@ Examples:
...
+ (i.e. `OR` search).  e.g. `Friday Monday` will return employees whose tags contain `Friday` or `Monday`.
+
+Examples:
+* `c-tag-find friday` returns all employees whose tags contain `friday`.
+* `c-tag-find friday parttime` returns all employees whose tag contain `friday`, `parttime`.<br>
```
Comment (WM71811) [inline docs/UserGuide.md]: Maybe tag could be be "tags" here?

File: docs/UserGuide.md (hunk 1 above)
Comment (yanbingtao) [reply docs/UserGuide.md]: Yes indeed, thank you for pointing out. I've changed it in the latest commit.

File: docs/UserGuide.md (hunk 2 above)
Comment (yanbingtao) [reply docs/UserGuide.md]: Thank you! I've changed it in the latest commit.

**Review by WM71811 — APPROVED** [review]
LGTM

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Pull request merged**